android/src/main/kotlin/dev/rexios/polar/PolarPlugin.kt

```diff
@@ -484,6 +519,88 @@ class PolarPlugin :
             })
             .discard()
     }
+
+    private fun doFirstTimeUse(call: MethodCall, result: Result) {
+        val arguments = call.arguments as List<*>
+        val identifier = arguments[0] as? String
+        val configJson = arguments[1] as? String
+
+        if (identifier == null || configJson == null) {
+            result.error(
+                "INVALID_ARGUMENTS",
+                "Expected identifier and config JSON",
+                null
+            )
+            return
+        }
+
+        val gson = Gson()
+        val config: FirstTimeUseConfig = try {
+            gson.fromJson(configJson, FirstTimeUseConfig::class.java)
+        } catch (e: Exception) {
+            result.error("JSON_PARSE_ERROR", "Invalid config JSON: ${e.message}", null)
+            return
+        }
+
+        val birthDate = try {
+            SimpleDateFormat("yyyy-MM-dd", Locale.getDefault()).parse(config.birthDate)
+        } catch (e: Exception) {
+            result.error("INVALID_DATE", "Could not parse birthDate: ${e.message}", null)
+            return
+        }
+
+        val genderEnum = try {
+            PolarEnumMapper.genderFromString(config.gender)
+        } catch (e: IllegalArgumentException) {
+            result.error("INVALID_GENDER", e.message, null)
+            return
+        }
+
+        val typicalDayEnum = PolarEnumMapper.typicalDayFromInt(config.typicalDay)
+
+        val ftuConfig = PolarFirstTimeUseConfig(
```

Comment:
You should be able to use gson directly with this class and not have to do any of the above mapping